pip/vcs/__init__.py

```diff
@@ -327,7 +327,10 @@ def run_command(self, cmd, show_stdout=True, cwd=None,
             # errno.ENOENT = no such file or directory
             # In other words, the VCS executable isn't available
             if e.errno == errno.ENOENT:
-                raise BadCommand('Cannot find command %r' % self.name)
+                raise BadCommand(
+                    'Cannot find command %r - do you have '
+                    '%r installed and in your '
+                    'PATH?' % self.name % self.name)
```

Comment:
You should not chain ```%``` operators like this. To supply multiple arguments to the ```%``` operator you need to supply a tuple on the RHS. But in this case I think that repeating the VCS command name is unnecessary - I'd just use "it" in place of the second ```%r```. If you prefer the repeat, though, that's fine (just implement it correctly, please).

One question - did you test this? As it stands the code will fail. I was considering asking you to write a test for this, but it seemed like a lot of additional work for a simple change. But I'm now wondering.